testsuite/MDAnalysisTests/coordinates/test_xdr.py

```diff
@@ -25,257 +25,209 @@
-        self.tmpdir = tempdir.TempDir()
-        self.outfile = self.tmpdir.name + '/xdr-reader-test' + ext
-
-    def tearDown(self):
-        try:
-            os.unlink(self.outfile)
-        except:
-            pass
-        del self.universe
-        del self.tmpdir
+    prec = 3
 
+    @pytest.fixture(scope='class')
+    def universe(self):
+        return mda.Universe(GRO, self.filename, convert_units=True)
 
     def test_flag_convert_lengths(self):
         assert_equal(mda.core.flags['convert_lengths'], True,
                      "MDAnalysis.core.flags['convert_lengths'] should be "
                      "True by default")
 
+    def test_rewind_xdrtrj(self, universe):
+        universe.trajectory.rewind()
+        assert_equal(universe.coord.frame, 0, "rewinding to frame 1")
 
-    def test_rewind_xdrtrj(self):
-        self.trajectory.rewind()
-        assert_equal(self.ts.frame, 0, "rewinding to frame 1")
-
-
-    def test_next_xdrtrj(self):
-        self.trajectory.rewind()
-        self.trajectory.next()
-        assert_equal(self.ts.frame, 1, "loading frame 1")
+    def test_next_xdrtrj(self, universe):
+        universe.trajectory.rewind()
+        universe.trajectory.next()
+        assert_equal(universe.coord.frame, 1, "loading frame 1")
 
+    def test_jump_xdrtrj(self, universe):
+        universe.trajectory[4]  # index is 0-based and frames are 0-based
+        assert_equal(universe.coord.frame, 4, "jumping to frame 4")
 
-    def test_jump_xdrtrj(self):
-        self.trajectory[4]  # index is 0-based and frames are 0-based
-        assert_equal(self.ts.frame, 4, "jumping to frame 4")
-
-
-    def test_jump_lastframe_xdrtrj(self):
-        self.trajectory[-1]
-        assert_equal(self.ts.frame, 9,
+    def test_jump_lastframe_xdrtrj(self, universe):
+        universe.trajectory[-1]
+        assert_equal(universe.coord.frame, 9,
                      "indexing last frame with trajectory[-1]")
 
-
-    def test_slice_xdrtrj(self):
-        frames = [ts.frame for ts in self.trajectory[2:9:3]]
+    def test_slice_xdrtrj(self, universe):
+        frames = [ts.frame for ts in universe.trajectory[2:9:3]]
         assert_equal(frames, [2, 5, 8], "slicing xdrtrj [2:9:3]")
 
-
-    def test_reverse_xdrtrj(self):
-        frames = [ts.frame for ts in self.trajectory[::-1]]
+    def test_reverse_xdrtrj(self, universe):
+        frames = [ts.frame for ts in universe.trajectory[::-1]]
         assert_equal(frames, list(range(9, -1, -1)), "slicing xdrtrj [::-1]")
 
-
-    def test_coordinates(self):
-        ca_nm = np.array([[6.043369675, 7.385184479, 1.381425762]],
-                         dtype=np.float32)
+    def test_coordinates(self, universe):
+        ca_nm = np.array(
+            [[6.043369675, 7.385184479, 1.381425762]], dtype=np.float32)
         # coordinates in the base unit (needed for True)
         ca_Angstrom = ca_nm * 10.0
-        U = self.universe
+        U = universe
```

Comment:
This aliasing seems superfluous, and further down you use `universe` instead of `U`. I would just use `universe` everywhere.